pkg/eventmonitor/consumer.go

```diff
@@ -8,15 +8,15 @@ package eventmonitor
 
 import "github.com/DataDog/datadog-agent/pkg/security/probe"
 
-// EventConsumer event consumer
-type EventConsumer interface {
-	probe.EventConsumerInterface
+// EventConsumerHandler provides an interface for event consumer handlers
```

Comment:
I would add to these comments the relationship that these two interfaces have with one another (AFAIK, the `EventConsumer` starts/stops the `EventConsumerHandler`).